game/scripts/npc/abilities/legion_commander_duel.txt

```diff
@@ -27,7 +27,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "50.0"
+    "AbilityCooldown"                                     "35.0" //OAA
```

Comment:
I know this change is better for the competitive but it will destroy pubs. So this will get through only if I lose my mind or if I stop playing OAA.